### PR TITLE
storage: Don't leave some jobs invisible

### PR DESCRIPTION
**Pull request by mvollmer**
- [x] #8062

### CODE REVIEW

**Comment by petervo** [comment]
Needs rebase

File: pkg/storaged/jobs-panel.jsx [hunk 1]
```diff
@@ -145,11 +145,25 @@ export class JobsPanel extends React.Component {
             return false;
         }
 
-        var jobs = Object.keys(client.jobs).filter(job_is_stable).sort(cmp_job);
+        var jobs = [ ];
+        var have_reminder = false;
+        for (var p in client.jobs) {
+            console.log(p);
```
Comment (petervo) [inline pkg/storaged/jobs-panel.jsx]: Probably don't want to leave this in, but i'll remove when merging.

Comment (mvollmer) [reply pkg/storaged/jobs-panel.jsx]: Yes, thanks!

Comment (mvollmer) [reply pkg/storaged/jobs-panel.jsx]: I have removed that line.

**Pull request closed without merging**